tools/clang/unittests/HLSL/CompilerTest.cpp

```diff
@@ -2812,6 +2813,32 @@ TEST_F(CompilerTest, CompileThenCheckDisplayIncludeProcess) {
 
 }
 
+TEST_F(CompilerTest, CompileThenPrintTimeReport) {
```

Comment:
Should we also have a FileChecker test to validate that the output gets embedded in the disassembly output?